python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -1004,3 +1073,33 @@ def loadSubExposures(self, bbox, statsCtrl, tempExpRefList, imageScalerList, spa
             exposure.image.array[(exposure.mask.array & statsCtrl.getAndMask()) > 0] = 0.
             subExposures[tempExpRef.dataId["visit"]] = exposure
         return subExposures
+
+    def selectCoaddPsf(self, templateCoadd, tempExpRefList):
+        """Compute the PSF of the coadd from the exposures with the best seeing.
+
+        Parameters
+        ----------
+        templateCoadd : `lsst.afw.image.ExposureF`
+            The initial coadd exposure before accounting for DCR.
+        tempExpRefList : `list` of `lsst.daf.persistence.ButlerDataRef`
```

Comment:
Can you just call these `warpRefList` instead?

Reply:
That's a relatively recent name change I hadn't seen. I'll change it.